internal/testing/releasehelper.go

```diff
@@ -132,24 +153,32 @@ func runOnGomod(path string) {
 				rel, _ = filepath.Split(rel)
 			}
 
-			if *doReplace {
-				cmdCheck(fmt.Sprintf("go mod edit -replace=%s=%s %s", r.Path, rel, gomodPath))
-			} else if *doDropReplace {
-				cmdCheck(fmt.Sprintf("go mod edit -dropreplace=%s %s", r.Path, gomodPath))
-			}
+			reqHandler(gomodPath, r.Path, rel)
 		}
 	}
 }
 
 func main() {
-	flag.Usage = func() {
-		fmt.Fprintf(flag.CommandLine.Output(), helpText)
-		flag.PrintDefaults()
+	if len(os.Args) < 2 {
+		printHelp()
+		os.Exit(0)
 	}
 
-	flag.Parse()
-	if *doReplace && *doDropReplace {
-		log.Fatal("Expected only one of -replace/-dropreplace")
+	var handlerFunc reqHandlerFunc
+	switch os.Args[1] {
+	case "help":
+		printHelp()
+		os.Exit(0)
+	case "addreplace":
+		handlerFunc = func(gomodPath string, mod string, modPath string) {
```

Comment:
Nit: `func(gomodPath, mod, modPath string)` here and below.

Reply:
Done